core/cmd/local_client.go

```diff
@@ -908,9 +909,12 @@ func dumpSchema(dbURL url.URL) (string, error) {
 		"pg_dump", args...,
 	)
 
+	stderr := &bytes.Buffer{}
+	cmd.Stderr = stderr
+
 	schema, err := cmd.Output()
 	if err != nil {
-		return "", fmt.Errorf("failed to dump schema: %v", err)
+		return "", fmt.Errorf("failed to dump schema: %v\n%s", err, stderr)
```

Comment:
This alternative used in the backup might read a little bit cleaner?
https://github.com/smartcontractkit/chainlink/blob/13208bb2e09e6a6e61febf7bb89a4881b960eec9/core/services/periodicbackup/backup.go#L196-L198
Or if that doesn't always report full info, then maybe it should be aligned with this one?
```go
	// Stderr holds a subset of the standard error output from the
	// Cmd.Output method if standard error was not otherwise being
	// collected.
	//
	// If the error output is long, Stderr may contain only a prefix
	// and suffix of the output, with the middle replaced with
	// text about the number of omitted bytes.
```